app.js

```diff
@@ -0,0 +1,48 @@
+const nav = document.getElementById('navbar__list');
+const sections = document.querySelectorAll('section');
+
+
+//Build the navigation menu
+const BuildTheNavigationMenu = () => {
+  let navUI = ``
+  for (i = 0; i < sections.length; i++) {
```

Comment:
in app.js 
I think it would be better to declare the variable I with let data type.

Reply:
- [ **OPTIONAL** ] The inline comment helps to directly point out the code where the changes are required. So specifying the file name seems redundant. Please kindly remove it. 👍🏽 
 - [ **OPTIONAL** ] It would have been nice if you could let the author know about the reason the change was requested it helps them to understand the requested changes better. 💯 
 - [ **OPTIONAL** ] How about using some emojis instead of just the text? 😉